compiler/parser/ceda/ast2shell.py

```diff
@@ -333,7 +333,7 @@ def string_of_arg_char (c, is_quoted=False):
         ## Chars to escape unconditionally
         chars_to_escape = ["'", '"', '`', '(', ')', '{', '}', '$', '!', '&', '|', ';']
```

Comment:
I suspect it would be safe move everything but `"` into `chars_when_no_quotes`, but no sense changing anything if it works.

Reply:
Great, I left a comment about that!